doc/src/build/sui-local-network.md

```diff
@@ -281,6 +281,11 @@ To resolve this, switch or update to Node.js 18 and then try again.
 
 For additional information about example data for testing, see [https://github.com/MystenLabs/sui/tree/main/sdk/typescript#testing](https://github.com/MystenLabs/sui/tree/main/sdk/typescript#testing).
 
+When [installing sui-test-validator](#Install Sui), if you do not have libpq installed, you might see the following message:
```

Comment:
@randall-Mysten This troubleshooting step is only applicable to the installation section, feel free to move it above if you think it's more appropriate

Reply:
I did move it. Should libq be added to prereqs for Sui or is it specific to the sui-test-validator only

Reply:
specific to the sui-test-validator only